### PR TITLE
added routefusion to the payments section

### PR DESCRIPTION
**Pull request by txssseal**
added Routefusion to the payments section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -683,6 +683,7 @@ Table of Contents
   * [vatlayer.com](https://vatlayer.com/) — Instant VAT number validation and EU VAT rates API, free 100 API requests/month
   * [fraudlabspro.com](https://www.fraudlabspro.com) — Help merchants to prevent payment fraud and chargebacks. Free Micro Plan available with 500 queries/month.
   * [currencystack.io](https://currencystack.io/) — Production-ready real-time exchange rates for 154 currencies.
+  * [routefusion.co](https://routefusion.co/) — Developer focused cross borders bank-to-bank payments API.  Programatically send money to 180+ countries.  Go live and begin transacting cross border bank to bank payment in your app in under a month with 0 cost and risk.
```
Comment (ripienaar) [inline README.md]: I do not see any information at all about costs associated with this service? Someone is paying, how and when?

Comment (txssseal) [reply README.md]: To develop and release code into production is completely free.  We make our money off of the exchange rate spreads, just like any cross border app.  The biggest difference is we are honest with the process, do not charge a monthly fee, or charge any type of integration fee.  We see ourselves as helping developers actually build new and innovative solutions without making them pay for it, like all of the competitors.

**Comment by ripienaar** [comment]
Is this made clear anywhere on the site?

**Comment by txssseal** [comment]
@ripienaar yes, if you signup you get developer keys and can begin developing asap.  When your application is done you can submit to promote your keys to prod.  Its a bit of process to go to production because of compliance with cross border payments, but we are very transparent with the entire process.

**Note**: We should actually do a better job on our splash page of describing how to go from sandbox to production.

**Comment by ripienaar** [comment]
I can only add to the list what I can verify in the site as free etc. 

So once you got clear info somewhere ping here and we can merge. Myself and the many people who regularly review and maintain this list have to be able to find the qualifying information. We don’t have time to reach out to owners or do signups to find info so it has to be on the site. Sorry that’s just how it is

**Comment by txssseal** [comment]
@ripienaar that makes sense, will talk to the team.  Thanks for the transparency.

**Comment by ripienaar** [comment]
closing this one - feel free to reopen when things are clearer

**Pull request closed without merging**